app/includes/header.test.js

```diff
@@ -94,28 +94,6 @@ describe('header', () => {
       }));
     });
 
-    it('should render the covid19 global warning if feature flag set', componentTester(setup, (harness) => {
-      const context = {
-        config: {
-          showCovid19: 'true',
-        },
-      };
-
-      harness.request(context, ($) => {
-        const globalAlert = $('[data-test-id="covid19-global-alert"]');
-        const title = globalAlert.find('div').find('h2');
-        const paragraph = globalAlert.find('div').find('p');
-        const vaccinationsLink = paragraph.find('[data-test-id="vaccinations"]');
-        const covid19Link = paragraph.find('[data-test-id="covid19"]');
-
-        expect(globalAlert.hasClass('bc-c-global-alert')).toEqual(true);
-        expect(title.text().trim()).toEqual('Coronavirus (COVID-19)');
-        expect(paragraph.text().trim()).toEqual('View Catalogue Solutions that help with coronavirus by organising vaccinations or reducing visits to GP practices.');
-        expect(vaccinationsLink.attr('href')).toEqual('/solutions/vaccinations');
-        expect(covid19Link.attr('href')).toEqual('/solutions/covid19');
-      });
-    }));
-
```

Comment:
As we're still rendering the banner based on the same feature flag is it not worth retaining this test but tweaking the assertions?